### PR TITLE
Game settings

### PR DESCRIPTION
**Pull request by Vlad-coder-678**
https://trello.com/c/FVeuOWio/27-create-issueslice-gamecardsslice-and-gamesettingslice

### CODE REVIEW

File: poker/src/components/CardStoryTypeShort/CardStoryTypeShort.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+import styles from './CardStoryTypeShort.module.scss';
+
+interface Props {
+  label: null | string;
```
Comment (Evgenia-cyber) [inline poker/src/components/CardStoryTypeShort/CardStoryTypeShort.tsx:5]: Please, change type label

**Review by Evgenia-cyber — APPROVED** [review]

File: poker/src/components/CardGameLobby/CardGameLobby.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+const CardGameLobby: FC<Props> = ({ card, isNew }) => {
+  const [isVisibleInput, setIsVisibleInput] = useState(false);
+  const dispatch = useDispatch();
+
+  const handleToggleVisible = (): void => {
+    setIsVisibleInput(!isVisibleInput);
+  };
+
+  const handleCreateCard = (): void => {
+    dispatch(createGC({ id: card.id, value: Number(card.value) >= 1000 ? '1000' : card.value }));
```
Comment (AnAtoliy-AA) [inline poker/src/components/CardGameLobby/CardGameLobby.tsx:26]: const cardValue = Number(card.value) >= 1000 ? '1000' : card.value;

dispatch(createGC({ id: card.id, value: cardValue  }));

File: poker/src/components/CardGameLobby/CardGameLobby.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  };
+
+  const handleChangeValue = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    const v = Number(e.target.value) >= 1000 ? '1000' : e.target.value;
```
Comment (AnAtoliy-AA) [inline poker/src/components/CardGameLobby/CardGameLobby.tsx:30]: empty line

File: poker/src/components/CardIssue/CardIssue.tsx [hunk 4]
```diff
@@ -1,41 +1,76 @@
...
+    setIsModalOpen(true);
+  };
+
+  const handleFixedIssue = (): void => {
+    setOption(options[1]);
+    setIsModalOpen(true);
+  };
+
+  const handleRemoveIssue = (): void => {
+    setOption(options[2]);
```
Comment (AnAtoliy-AA) [inline poker/src/components/CardIssue/CardIssue.tsx:38]: magic number

**Review by AnAtoliy-AA — APPROVED** [review]

File: poker/src/components/CardGameLobby/CardGameLobby.tsx (hunk 2 above)
Comment (AnAtoliy-AA) [reply poker/src/components/CardGameLobby/CardGameLobby.tsx:26]: 1000 magic number

File: poker/src/components/CardStoryTypeShort/CardStoryTypeShort.tsx (hunk 1 above)
Comment (Vlad-coder-678) [reply poker/src/components/CardStoryTypeShort/CardStoryTypeShort.tsx:5]: fixed

File: poker/src/components/CardGameLobby/CardGameLobby.tsx (hunk 2 above)
Comment (Vlad-coder-678) [reply poker/src/components/CardGameLobby/CardGameLobby.tsx:26]: fixed

File: poker/src/components/CardIssue/CardIssue.tsx (hunk 4 above)
Comment (Vlad-coder-678) [reply poker/src/components/CardIssue/CardIssue.tsx:38]: fixed

**Pull request merged**